lua/user/plugins.lua

```diff
@@ -282,58 +282,55 @@ M.config = function()
       config = function()
         require("user.ntest").config()
       end,
-      requires = {
-        { "nvim-neotest/neotest-go" },
-        { "nvim-neotest/neotest-python" },
+      dependencies = {
         { "nvim-neotest/neotest-plenary" },
-        { "rouge8/neotest-rust" },
       },
-      -- opt = true,
-      -- event = { "BufEnter *_test.*,*_spec.*,test_*.*" },
-      disable = not (lvim.builtin.test_runner.active and lvim.builtin.test_runner.runner == "neotest"),
+      enabled = (lvim.builtin.test_runner.active and lvim.builtin.test_runner.runner == "neotest"),
     },
+    { "nvim-neotest/neotest-go", event = { "BufEnter *.go" } },
+    { "nvim-neotest/neotest-python", event = { "BufEnter *.py" } },
+    { "nvim-rust/neotest-rust", event = { "BufEnter *.rs" } },
```

Comment:
@abzcoding Ping me If you don't like these, and I'll remove these

Reply:
could it not be loaded on command? with? or did I misunderstood this from the lazy.nvim site

Reply:
Well, I think you're onto something. But does neotest-go and neatest-python have commands? Or are they simply "plugins" that plugs into neotest?

Reply:
Good question! I would use neotest as a command, but this would load all the neotest-plugins.
The BufEnter may be the best solution, but it means neotest will load on .rs, even if you did not intend to run neotest.
Maybe @folke have an idea?

Reply:
it should be `rouge8/neotest-rust`

Reply:
Fixed in d218f26